<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportTableDialog</class>
 <widget class="QDialog" name="ExportTableDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>541</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>665</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Export Table...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0" colspan="4">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>280</width>
         <height>250</height>
        </size>
       </property>
       <property name="title">
        <string>Objects</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QListView" name="listView_1">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::MultiSelection</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QPushButton" name="objselectAllButton">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>130</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>Select All</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="objinvertselectionButton">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>130</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>Invert Selection...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QComboBox" name="objSelectByLabel">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>130</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="objunselectAllButton">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>130</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>Unselect All</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="minimumSize">
        <size>
         <width>280</width>
         <height>250</height>
        </size>
       </property>
       <property name="title">
        <string>Variables</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QListView" name="listView_2">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::MultiSelection</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QPushButton" name="varselectAllButton">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>130</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>Select All</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="varinvertselectionButton">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>130</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>Invert Selection...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QComboBox" name="varSelectByLabel">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>130</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="varunselectAllButton">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>130</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>Unselect All</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>291</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2" colspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>159</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Separator:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="separator">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>112</width>
         <height>23</height>
        </size>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>TAB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SPACE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>,</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>;</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>&lt;CUSTOM&gt;</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="saveButton">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>Save As...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLineEdit" name="filetosave">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>23</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>434</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="cancelButton">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="okButton">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>horizontalSpacer_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
